cmd/format-disk-cache.go

```diff
@@ -372,7 +373,19 @@ func migrateData(ctx context.Context, c *diskCache, oldfile, destDir string) err
 	if err != nil {
 		return err
 	}
-	_, err = c.bitrotWriteToCache(ctx, destDir, readCloser, st.Size())
+	var reader io.Reader
+	reader = readCloser
```

Comment:
CI static analyzer is failing here.  
```suggestion
        var reader io.Reader = readCloser
```